packages/server/src/tests/amp/ampEpicModels.ts

```diff
@@ -20,6 +20,7 @@ export interface AMPEpic {
     highlightedText?: string;
     cta: AMPCta;
     ticker?: AMPTicker;
+    choiceCards?: boolean;
```

Comment:
I think we only need this on the variant model?

Reply:
@TomPretty the `AmpEpicTestVariant` model is for the data from the tool, whereas the `AMPEpic` model is for the data sent to the frontend for the epic. The `AMPEpic` data comes from the variant, somewhat confusingly.

Reply:
Makes sense - I guess names are a little confusing here!

Reply:
Yeah, at some point I'm going to try to do a big refactor of all the AMP epic stuff!